### PR TITLE
Add networkx calendar

### PR DESCRIPTION
**Pull request by dschult**

### CODE REVIEW

File: calendars/networkx.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: NetworkX Community Calendar
+timezone: America/Los_Angeles
+events:
+  - summary: NetworkX Community Call (Americas/Oceania)
+    description: |
+      The NetworkX Community Call. 
+
+      Meeting notes: https://hackmd.io/ea2IhUuqSrG4kM9tokXuEw
```
Comment (tupui) [inline calendars/networkx.yaml:8]: FYI you can customise the link to have a user friendlier name
Screenshot 2022-06-07 at 00 09 38
.

Comment (jarrodmillman) [reply calendars/networkx.yaml:8]: For some reason, I don't get that option.
screenshot
So let's merge this and sort out a better name later. Maybe we need to move the hackmd to the scientific-python hackmd.

Comment (jarrodmillman) [reply calendars/networkx.yaml:8]: I will follow-up on this in a future PR.

Comment (tupui) [reply calendars/networkx.yaml:8]: Sure OC.

Still strange, and you are the owner of the file? Also it's strange that the address is `https://hackmd.io/@ea2IhUuqSrG4kM9tokXuEw` and not something like `https://hackmd.io/@jarrodmillman/...`.

I can do that on any files of my own, they don't need to be in the org. Maybe I clicked on some options to validate my profile or something.

**Pull request merged**